lib/request.js

```diff
@@ -422,7 +422,7 @@ module.exports = {
    * When `app.proxy` is `true`, parse
    * the "X-Forwarded-For" ip address list.
    *
-   * For example if the value were "client, proxy1, proxy2"
+   * For example if the values were "client, proxy1, proxy2"
```

Comment:
I disagree, a string is one value.

Reply:
Makes sense, I adjusted it from "the values were" to the "the value was", squashed the commits, and pushed.